### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by letzya**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [ ] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
     - [ ] I have manually updated the README(s)/documentation accordingly.
     - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
     - [ ] If new config option added, ensure that it can be set via ENV variable
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] When updating library version must provide reason/explanation for this update.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
    - [ ] `go fmt -s`
    - [ ] `go vet`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,52 +32,77 @@ import (
...
+## Error Handling
+The library provides several error types:
+
+`ErrNilConfig`: Returned when a nil configuration struct is provided.
+`ErrEmptyStruct`: Returned when an empty struct is provided.
+`ErrInvalidObjectType`: Returned when the object is not of struct type.
+Contributing
+We welcome contributions! Please see our contribution guidelines for details.
+
+By incorporating these elements, your README will be more accessible and informative for users and contributors alike.
```
Comment (mativm02) [inline README.md:95]: Remove this line

File: README.md [hunk 2]
```diff
@@ -23,52 +32,77 @@ import (
...
 
-You can pass query parameters `field` or `env` to the endpoints to retrieve specific config fields or specific environment variables.
 
 ## Limitations
 
-- Only exported fields in struct are parsed
+- Only exported fields in go struct are parsed
```
Comment (mativm02) [inline README.md:100]: "go" should start with an uppercase -> "Go"

File: README.md [hunk 3]
```diff
@@ -23,52 +32,77 @@ import (
...
+`/envs`: Exposes environment variables mapped from the config object.
+
+
+## Error Handling
+The library provides several error types:
+
+`ErrNilConfig`: Returned when a nil configuration struct is provided.
+`ErrEmptyStruct`: Returned when an empty struct is provided.
+`ErrInvalidObjectType`: Returned when the object is not of struct type.
+Contributing
```
Comment (mativm02) [inline README.md:92]: Shouldn't this start with "##"?

File: README.md [hunk 4]
```diff
@@ -23,52 +32,77 @@ import (
...
 	}
 
 	http.HandleFunc("/config", v.JSONHandler)
 	http.HandleFunc("/envs", v.EnvsHandler)
-	log.Println("Running server on port :8080")
 	err = http.ListenAndServe(":8080", nil)
 	if err != nil {
 		log.Fatal("ListenAndServe: ", err)
 	}
+	log.Println("Running server on port :8080")
```
Comment (mativm02) [inline README.md:66]: WIll this be printed? If I remember correctly, this line:
```
err = http.ListenAndServe(":8080", nil)
```
blocks the execution of the program at that step if I'm not wrong.

File: README.md [hunk 5]
```diff
@@ -1,17 +1,26 @@
...
+
+- Parse struct-based configurations.
+- Expose configuration fields and environment variables as JSON.
+- Simple HTTP handlers to expose config and envs.
+- Secure: Automatically obfuscate sensitive data.
 
 ## Installation
 
-To install the struct_viewer package, use the go get command:
+To install the *struct_viewer* package, use the go get command:
```
Comment (mativm02) [inline README.md:14]: go get could be written with backticks, like `go get`

**Review by mativm02 — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (letzya) [inline README.md:14]:
```suggestion
To install the *struct_viewer* package, use the `go get` command:
```

File: README.md (hunk 4 above)
Comment (letzya) [inline README.md:66]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -57,18 +68,40 @@ func main() {
...
+`/envs`: Exposes environment variables mapped from the config object.
+
+
+## Error Handling
+The library provides several error types:
+
+`ErrNilConfig`: Returned when a nil configuration struct is provided.
+`ErrEmptyStruct`: Returned when an empty struct is provided.
+`ErrInvalidObjectType`: Returned when the object is not of struct type.
+Contributing
```
Comment (letzya) [inline README.md:91]:
```suggestion
## Contributing
```

File: README.md [hunk 7]
```diff
@@ -57,18 +68,40 @@ func main() {
...
+We welcome contributions! Please see our contribution guidelines for details.
+
+By incorporating these elements, your README will be more accessible and informative for users and contributors alike.
 
-You can pass query parameters `field` or `env` to the endpoints to retrieve specific config fields or specific environment variables.
 
 ## Limitations
 
-- Only exported fields in struct are parsed
+- Only exported fields in go struct are parsed
```
Comment (letzya) [inline README.md:99]:
```suggestion
- Only exported fields in Go struct are parsed
```

File: README.md [hunk 8]
```diff
@@ -57,18 +68,40 @@ func main() {
...
+## Error Handling
+The library provides several error types:
+
+`ErrNilConfig`: Returned when a nil configuration struct is provided.
+`ErrEmptyStruct`: Returned when an empty struct is provided.
+`ErrInvalidObjectType`: Returned when the object is not of struct type.
+## Contributing
+We welcome contributions! Please see our contribution guidelines for details.
+
+By incorporating these elements, your README will be more accessible and informative for users and contributors alike.
```
Comment (letzya) [inline README.md:94]:
```suggestion
```

**Review by mativm02 — APPROVED** [review]

**Pull request merged**